### DISCUSSION TITLE
using fintenued model from hugging face

### DISCUSSION THREAD

**Discussion by Abhishek-cmd13** [NavodPeiris/speechlib#41] [Ideas]
Hey is there a way I can use a fine tuned model of whisper that is available on hugging face

**Comment by NavodPeiris** [comment]
hi, i made the library simple so anyone can use it without writing much code. If you want a specific whisper model instead of standard one. then library level code has to be modified. however, this can be added as a user input so users can pick their model. i will give it a try.

**Reply by Abhishek-cmd13** [reply]
Can you please , do it.

I gave past 2 days so I was replacing the model like you did for sinhala but I am getting a empty transcript , It gives transcription done Time take 0 seconds
